package.json

```diff
@@ -49,8 +45,8 @@
     "mqtt": "^2.18.8",
     "multer": "^1.4.1",
     "mustache": "^3.0.1",
-    "mysql": "^2.16.0",
-    "mysql2": "^1.6.5",
+    "mysql": "^2.18.1",
```

Comment:
Mysql dependency needs to still be here, as better-queue-sql only supports mysql, not mysql2. Ideally the support should be applied as a feature to the better-queue-sql library but as we are leaving the node environment anyways in the future, I believe this is the best we can do for now.